command/connect/envoy/envoy.go

```diff
@@ -422,7 +423,7 @@ func (c *cmd) templateArgs() (*BootstrapTplArgs, error) {
 	if strings.HasPrefix(strings.ToLower(c.grpcAddr), "https://") {
 		useTLS = true
 	} else if useSSLEnv := os.Getenv(api.HTTPSSLEnvName); useSSLEnv != "" {
-		if enabled, err := strconv.ParseBool(useSSLEnv); err != nil {
+		if enabled, err := strconv.ParseBool(useSSLEnv); err == nil {
```

Comment:
Found a little bug as well.

Reply:
Good catch. I still thought this was wrong for a second but you're right...